src/gateway/mod.rs

```diff
@@ -3,7 +3,7 @@
 //!
 //! A shard is an interface for the lower-level receiver and sender. It provides
 //! what can otherwise be thought of as "sugar methods". A shard represents a
-//! single connection to Discord. If acting as a [`Bot`] user, you can make
+//! single connection to Discord. If acting as a [bot user], you can make
```

Comment:
Let's remove this "If acting as a bot user" text, the library no longer needs to make the differentiation.